CONTRIBUTING.md

```diff
@@ -138,6 +138,8 @@ and `python setup.py clean`. Then you can install in `develop` mode again.
 * A prerequisite to installing PyTorch is CMake. We recommend installing it with [Homebrew](https://brew.sh/)
 with `brew install cmake` if you are developing on MacOS or Linux system.
 * Our `setup.py` requires Python >= 3.6
+* If a commit is simple and doesn't affect any code (keep in mind that some docstrings contain code
+  that is used in tests), you can add `[skip ci]` to your commit message to skip all build / test steps.
```

Comment:
Is it only in the commit message? Could one have the [skip ci] in the PR title or body as well?

Reply:
I also see you experimenting with the capitalization. Maybe include a note about the casing if it's restricting?

Reply:
Ah I see it has to be the commit message: do you think it may be helpful to link https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/?

Reply:
Based on #55425 looks like anywhere in the commit message (subject or body) is fine, but updating the PR's title/body on GitHub has no effect: #55426